src/components/LanguageSelector/LanguageSelector.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react'
 import globe_icon from "../../assets/globe_icon.svg"
 import styles from "./LanguageSelector.module.css"
-
+import LanguagePopUp from '../LanguagePopUp/LanguagePopUp'
+import { useState, useEffect } from 'react'
 
 const LanguageSelector = () => {
+    const [isVisiable, setIsVisiable] = useState(false);
+    const handelLanguageSelectionClick = () => {
+        setIsVisiable(true);
+    }
+    const handelClose = () => {
+        setIsVisiable(false);
+    }
+    useEffect(()=>{
+        setIsVisiable(false);
+    },[])
+
```

Comment:
This useEffect seems unnecessary as the default value is set to 'false' in useState.